roles/network-multus/templates/cni-plugins.yml

```diff
@@ -1,4 +1,35 @@
 ---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sriov-nodes-config
+  namespace: {{ network_namespace }}
+data:
+  fallback-config: |
+    {
+        "resourceList":
+        [
+            {
+                "resourceName": "sriov",
+                "rootDevices": [],
```

Comment:
We should expose the current variable with the list of root devices here, so that setting it configures all nodes to use it. (We should later rethink how we configure these devices per node but it can wait.)

Reply:
Done